src/mon/PGMap.cc

```diff
@@ -751,10 +751,12 @@ void PGMapDigest::dump_pool_stats_full(
     if (verbose) {
       tbl.define_column("QUOTA OBJECTS", TextTable::RIGHT, TextTable::RIGHT);
       tbl.define_column("QUOTA BYTES", TextTable::RIGHT, TextTable::RIGHT);
-      tbl.define_column("DIRTY", TextTable::RIGHT, TextTable::RIGHT);
       tbl.define_column("USED COMPR", TextTable::RIGHT, TextTable::RIGHT);
       tbl.define_column("UNDER COMPR", TextTable::RIGHT, TextTable::RIGHT);
     }
+
+    if(verbose && is_tier()) {
```

Comment:
https://jenkins.ceph.com/job/ceph-pull-requests/75939/consoleFull#3136400332a811ea2-3e7b-466b-84b4-d13df7e35809

Reply:
> https://jenkins.ceph.com/job/ceph-pull-requests/75939/consoleFull#3136400332a811ea2-3e7b-466b-84b4-d13df7e35809

sorry, should have verified locally, earlier. 
bin/ceph df detail(before):
```
--- RAW STORAGE ---
CLASS     SIZE    AVAIL     USED  RAW USED  %RAW USED
hdd    101 GiB  100 GiB  1.0 GiB   1.0 GiB       1.04
TOTAL  101 GiB  100 GiB  1.0 GiB   1.0 GiB       1.04

--- POOLS ---
POOL                   ID  PGS  STORED  (DATA)  (OMAP)  OBJECTS    USED  (DATA)  (OMAP)  %USED  MAX AVAIL  QUOTA OBJECTS  QUOTA BYTES  DIRTY  USED COMPR  UNDER COMPR
device_health_metrics   1    1     0 B     0 B     0 B        0     0 B     0 B     0 B      0     99 GiB            N/A          N/A      0         0 B          0 B
test                    2   32  60 MiB  60 MiB     0 B       11  60 MiB  60 MiB     0 B   0.06     99 GiB            N/A          N/A     11         0 B          0 B
```
after:

```
[ideepika@vossi03 build]$ bin/ceph df detail
*** DEVELOPER MODE: setting PATH, PYTHONPATH and LD_LIBRARY_PATH ***
2021-05-20T12:04:31.671+0000 7f32012a6700 -1 WARNING: all dangerous and experimental features are enabled.
2021-05-20T12:04:31.693+0000 7f32012a6700 -1 WARNING: all dangerous and experimental features are enabled.
--- RAW STORAGE ---
CLASS     SIZE    AVAIL     USED  RAW USED  %RAW USED
hdd    101 GiB  100 GiB  1.0 GiB   1.0 GiB       0.99
TOTAL  101 GiB  100 GiB  1.0 GiB   1.0 GiB       0.99

--- POOLS ---
POOL                   ID  PGS  STORED  (DATA)  (OMAP)  OBJECTS  USED  (DATA)  (OMAP)  %USED  MAX AVAIL  QUOTA OBJECTS  QUOTA BYTES  DIRTY  USED COMPR  UNDER COMPR
device_health_metrics   1    1     0 B     0 B     0 B        0   0 B     0 B     0 B      0     99 GiB            N/A          N/A    N/A
test                    2   32     0 B     0 B     0 B        0   0 B     0 B     0 B      0     99 GiB            N/A          N/A    N/A
```